src/agent.spec.ts

```diff
@@ -0,0 +1,549 @@
+import { Finding, ethers, FindingSeverity, FindingType } from "forta-agent";
+import { TestTransactionEvent } from "forta-agent-tools/lib/test";
+import { createAddress, createTransactionHash } from "forta-agent-tools/lib/utils";
+import agent from "./agent";
+
+const { handleTransaction } = agent;
+
+import {
+  NETHERMIND_DEPLOYER_ADDRESS,
+  AGENT_REGISTRY_ADDRESS,
+  CREATE_AGENT_ABI,
+  UPDATE_AGENT_ABI,
+  DISABLE_AGENT_ABI,
+} from "./constants";
+
+export const iface = new ethers.utils.Interface([
```

Comment:
```suggestion
export const testIFace = new ethers.utils.Interface([
```
After updated to use test constants.